apps/cow-fi/services/cms/index.ts

```diff
@@ -146,9 +148,97 @@ export async function getArticles({
     throw error
   }
 
+  // If fetchAll is true and there are more pages, recursively fetch them
+  if (
```

Comment:
I see you changed `searchArticles` function, so we can use Strapi API for searching.
I would expect we don't need recursive `getArticles` fetching. Are there any reasons left?

Reply:
Thanks for flagging. Addressed now.